index.ios.js

```diff
@@ -19,6 +19,7 @@ import thunk from 'redux-thunk';
 import createLogger from 'redux-logger';
 import promise from 'redux-promise';
 const SideMenu = require('react-native-side-menu');
+const Firebase = require('firebase');
```

Comment:
It looks like we are repeating ourselves here by using Firebase and setting up a ref later in the file, and also passing it as a prop to ViewContainer.

However, ViewContainer doesn't actually make use of that prop, and we also have separated out setting up Firebase into another file. Let's get rid of references to firebase in this file, and import our firebase module where we use it, instead.